lib/camping.rb

```diff
@@ -1,7 +1,7 @@
 require "cam\ping/loads";E||="content-type";Z||="text/html"
 class Object;def meta_def m,&b;(class<<self;self
 end).define_method(m,&b) end;end
-module Camping;C=self;S=IO.read(__FILE__)rescue nil
+module Camping;C=self;S=IO.read(__FILE__)rescue()
```

Comment:
This is fantastic but needs to be reflected in `camping-unabridged.rb`. We gotta keep it in sync.

Reply:
You know what never mind. This isn't necessary to match the two.  APPROVED!